proxy/src/main/kotlin/net/rsprox/proxy/ProxyService.kt

```diff
@@ -688,6 +694,15 @@ public class ProxyService(
         }
     }
 
+    private fun loadJavConfig(rspsHost: String?): JavConfig {
+        val url = "http://${rspsHost ?: "oldschool.runescape.com"}/jav_config.ws"
```

Comment:
I'd rather `rspsHost` be a full url pointing to the jav_config file. This current method is too restrictive, forces a specific file name.